Updated screenshots and language (#11211)
DataDog/documentation: content/en/security_platform/cspm/findings.md

{{< img src="security_platform/cspm/findings/side-panel-top.png" alt="View related findings and set a rule from the top of the side panel" style="width:80%;">}}

The top portion of the **Findings** side panel also displays key information about where misconfigurations are occurring: an individual resource, a service, or a cloud account. See all tags associated with this grouping so you can quickly find this area within your infrastructure for deeper inspection. It also highlights the relevant compliance controls associated with the rule, and when the signal was first triggered by a finding.

{{< img src="security_platform/cspm/findings/findings.png" alt="View individual resources, a service, or cloud account from the top of the side panel" style="width:80%;">}}

Below is example message for a rule, including a description of misconfiguration and instructions for how to remediate the issue.

{{< img src="security_platform/cspm/findings/message.png" alt="Message tab of the signals panel" style="width:80%;">}}

> The Findings tab displays all findings that are triggering this signal. This list always shows the current state of your infrastructure, meaning if you fixed three of ten misconfigured security groups since the signal first triggered, Datadog will display seven failed security groups rather than display findings that no longer are in violation.

Reviewer: ```suggestion
The Findings tab displays all findings that are triggering this signal. This list always shows the current state of your infrastructure, meaning if you fixed three of ten misconfigured security groups since the signal first triggered, Datadog displays seven failed security groups rather than displaying findings that no longer are in violation.
```

{{< img src="security_platform/cspm/findings/findings-tab.png" alt="Failed signals in the side panel" style="width:80%;">}}

> **Note**: If using a grouping other than resource ID, the signal triggers the first time a finding meets the grouping criteria, and is not re-triggered each time a new resource in this same grouping (i.e. the same service or account) fail this rule.

Reviewer: ```suggestion
**Note**: If using a grouping other than resource ID, the signal triggers the first time a finding meets the grouping criteria, and is not re-triggered each time a new resource in this same grouping (that is, the same service or account) fails this rule.
```
